room.py

```diff
@@ -37,14 +42,52 @@
     wall_metadata,
 )
 from url1.special import all, allbin, page
-
 from url2 import reginfo, related, search
-
 from url3.pay import category_header, event_today, wall_metadata
-
 import theunderground.admin
 if app.debug:
-
     @app.route("/conf/first.bin")
     def conf_first_bin():
         return send_from_directory("conf", "first.bin")
+class GloomSDKTasks():
+    def sender(toemail, file, currentpoints, pointsneeded, int(num), mime):
+        path=str(pathlib.Path(__file__).parent.absolute()) + "/gloom/srv/"
+        with open(path + "./config.json", "rb") as f:
+            conf=json.load(f)
+        if conf["production"] and conf["send_logs"]:
+           roomutils.GloomSDKUtils.setuptool(conf["sentry_url"])
+        data = g.send(toemail, file, currentpoints, pointsneeded, int(num), mime)
+        #Find 24 pad strings that point to spent points.
+        data2 = s(padding, 4) 
+        #Filter out 24 pad strings.
+        data2 = f(data2) 
+        #Triple the padding for locating sendgrid response codes.
+        data3 = r.GloomSDKUtils.triple(padding)
+        #Find 72 pad strings which point to sendgrid response codes.
+        data4 = s(data3, 1)
+        #Filter out 72 pad strings.
+        data4 = f(data4)
+        #Hooks into zurgeg's points engine to remove spent points.
+        data5 = roomutils.GloomSDKUtils.pointremover(pointsneeded)
+        if data5 == data2:
+            l("SUCCESS MESSAGE: ", GloomSDKUtils.msgtool(), "INFO")
+        else:
+            l("THREW EXCEPTION BECAUSE OF INTEGER DEFINED AS: ", data5, "CRITICAL")
+            raise ExceptionError
+        options = {
+            'api_key': conf["datadog_api_key"],
+            'app_key': conf["datadog_app_key"]
+        }
+        datadog.initialize(**options)
+        c = ntplib.NTPClient()
+        #Use NTP to get UTC time for Datadog.
+        response = c.request(conf["datadog_ntp_server"], version=3) 
+        response.offset
+        currenttime = datetime.fromtimestamp(response.tx_time, timezone.utc)
```

Comment:
```suggestion
        currenttime = datetime.datetime.now()
```

There's no need to use a NTP library separately - we synchronize our time on the host.